exercises/diamond/description.md

```diff
@@ -1,25 +1,9 @@
 # Description
 
-The diamond kata takes as its input a letter, and outputs it in a diamond shape.
-Given a letter, it prints a diamond starting with 'A', with the supplied letter at the widest point.
+Your objective is to draw out a diamond using letters and spaces.
 
-## Requirements
-
-- The first row contains one 'A'.
-- The last row contains one 'A'.
-- All rows, except the first and last, have exactly two identical letters.
-- All rows have as many trailing spaces as leading spaces. (This might be 0).
-- The diamond is horizontally symmetric.
-- The diamond is vertically symmetric.
-- The diamond has a square shape (width equals height).
-- The letters form a diamond shape.
-- The top half has the letters in ascending order.
-- The bottom half has the letters in descending order.
-- The four corners (containing the spaces) are triangles.
-
-## Examples
-
-In the following examples, spaces are indicated by `·` characters.
+You're given a letter which signifies the widest point of the diamond.
```

Comment:
```suggestion
You're given a letter which should be at the widest part of the diamond.
```

Reply:
Yeah - I can't find a version I love

@Cool-Katt You ok with:
> You're given a letter, which represents the widest point of the diamond.
?

Reply:
@iHiD How about 
> You're given a letter, which should be at the widest point of the diamond.

Reply:
A tentative agree I suppose. 
I don't think the word 'represents' fits here quite nicely.  How about something like
> You're given a letter, which should correspond to the widest part of the diamond.
or 
> You're given a letter, which should depict the widest part of the diamond.

Reply:
So then maybe something like 
> You're given a letter, which should be at the middle two tips of the diamond.

Reply:
>Maybe "which tells you the..."?
How would the entire sentence look in that case?

Reply:
How about saying nothing, and perhaps try to simplify the requirements? Isn't it clear from the examples that the given letter always appears in the middle row?